### PR TITLE
Add-a-review components

### PR DESCRIPTION
**Pull request by thachdo90**
Implemented these components so far:
Overall rating
Do you recommend
Summary
Body
Upload photos
nickname:
email:

IN THE WORKS:
factors breakdown component
make this a popup, right now it sits at the very bottom of the page

### CODE REVIEW

File: client/src/components/RatingsReviews/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <span key={index}>
+          <input type="radio" id={index} name="rating" value={index} required onClick={changeRating} />
+          <label for={index}><img src={rating >= index ? fullStar : noStar} onMouseOver={() => tempRating(index)}/></label>
+        </span>
+      )
+    }
+  })
+
+  const countChar = () => {
+    let charCount = document.getElementById('RR-body') ? document.getElementById('RR-body').value.length : 0;
```
Comment (phok1012) [inline client/src/components/RatingsReviews/Form.jsx:36]: Do not know if this is possilbe, but can you have a parameter of event and just use event.target.value.length?

**Review by phok1012 — APPROVED** [review]
left one comment. Looks like you did a lot of research under FileReader(). I dont even know what that does. but your Form looks like a lot of work. Good Job !

**Review by dantan510 — APPROVED** [review]
wow great job! i can see how much effort you put into this. its great to see you're already trying to implement the more advanced react concepts

File: client/src/components/RatingsReviews/Form.jsx (hunk 1 above)
Comment (thachdo90) [reply client/src/components/RatingsReviews/Form.jsx:36]: Oh yea, that'll make it cleaner. Thanks!

**Pull request merged**